src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1662,6 +1664,55 @@ void linkItemToHoldTLRWithHoldShelfWhenCheckedInItemThenFulfilledWithSuccess(){
     assertThat(itemRepresentation.getJsonObject("status").getString("name"), is("Awaiting pickup"));
   }
 
+  @Test
+  void checkInItemWhenServiceHasChangedToNoPickupLocation(){
+    reconfigureTlrFeature(TlrFeatureStatus.NOT_CONFIGURED);
```

Comment:
No need to do this, `configurationsFixture.enableTlrFeature()` will take care of wiping existing configuration

Reply:
done